boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -447,6 +474,129 @@ unsafe fn setup() -> (
         apollo3::ble::Ble,
     ));
 
+    // Flash
+    let flash_ctrl_read_buf = static_init!(
+        [u8; apollo3::flash_ctrl::PAGE_SIZE],
+        [0; apollo3::flash_ctrl::PAGE_SIZE]
+    );
+    let page_buffer = static_init!(
+        apollo3::flash_ctrl::Apollo3Page,
+        apollo3::flash_ctrl::Apollo3Page::default()
+    );
+
+    let mux_flash = components::flash::FlashMuxComponent::new(&peripherals.flash_ctrl).finalize(
+        components::flash_mux_component_static!(apollo3::flash_ctrl::FlashCtrl),
+    );
+
+    // SipHash
+    let sip_hash = static_init!(
+        capsules_extra::sip_hash::SipHasher24,
+        capsules_extra::sip_hash::SipHasher24::new()
+    );
+    kernel::deferred_call::DeferredCallClient::register(sip_hash);
+
+    // TicKV
+    let tickv = components::tickv::TicKVComponent::new(
+        sip_hash,
+        mux_flash,                                             // Flash controller
+        (0x100000 - 0x28000) / apollo3::flash_ctrl::PAGE_SIZE, // Region offset (Last 0x28000 bytes of flash)
+        // Region Size, the final page doens't work correctly
+        0x28000 - apollo3::flash_ctrl::PAGE_SIZE,
```

Comment:
This seems tied to the space that we reserve in the linker script. Can you make this a dedicated `MEMORY` region in the linker script and export start and end symbols to it? That would make it more obvious what those numbers mean, and should be more reliable in the face of changes to the linker script.